docs/core/diagnostics/dotnet-trace.md

```diff
@@ -5,37 +5,44 @@ author: sdmaclea
 ms.author: stmaclea
 ms.date: 10/14/2019
 ---
-# Trace for performance analysis utility (`dotnet-trace`)
+# dotnet-trace performance analysis utility
 
 **This article applies to:** .NET Core 3.0 SDK and later versions
 
-## Installing `dotnet-trace`
+## Install dotnet-trace
 
-To install the latest release version of the `dotnet-trace` [NuGet package](https://www.nuget.org/packages/dotnet-trace), use the [dotnet tool install](../tools/dotnet-tool-install.md) command:
+Install `dotnet-trace` [NuGet package](https://www.nuget.org/packages/dotnet-trace) with the [dotnet tool install](../tools/dotnet-tool-install.md) command:
 
 ```dotnetcli
 dotnet tool install --global dotnet-trace
 ```
 
 ## Synopsis
 
-```console
+```dotnetcli
 dotnet-trace [-h, --help] [--version] <command>
 ```
 
 ## Description
 
-The `dotnet-trace` tool is a cross-platform CLI global tool that enables the collection of .NET Core traces of a running process without any native profiler involved. It's built around the cross-platform `EventPipe` technology of the .NET Core runtime. `dotnet-trace` delivers the same experience on Windows, Linux, or macOS.
+The `dotnet-trace` tool:
+
+* Is a cross-platform CLI global tool.
+* Enables the collection of .NET Core traces of a running process without a native profiler .
+* Is built around the cross-platform `EventPipe` technology of the .NET Core runtime.
```

Comment:
This fixes a lint error: `MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "* Is built around the cross-pl..."]`.
```suggestion
* Is built around the cross-platform `EventPipe` technology of the .NET Core runtime.

```